actionpack/actionpack.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "activesupport", version
 
+  s.add_dependency "nokogiri", ">= 1.8.5"
```

Comment:
This version was copied from [actiontext/actiontext.gemspec](https://github.com/rails/rails/blob/2fc46f95a5fcc6e80c080cc2a6fd6e48bc75115f/actiontext/actiontext.gemspec#L40).